controllers/dscinitialization/resources/authorino/deployment.injection.patch.tmpl.yaml

```diff
@@ -6,5 +6,7 @@ metadata:
 spec:
   template:
     metadata:
+      labels:
+        sidecar.istio.io/inject: "true"
       annotations:
```

Comment:
I think we can remove the annotations altogether here, the label should suffice for injection.

Reply:
i was thinking to keep the old "annoations" for a while, till the new authorino release is out then we can do the cleanups.
but from my local test, labels already serve the work to pass it down to pod, which means annotations is not really needed. 
i leave it to next week till Bartosz is back for a double check.

Reply:
i removed the "annotations" now.